WasatchNET/Spectrometer.cs

```diff
@@ -2763,6 +2772,54 @@ public virtual double[] getSpectrum(bool forceNew=false)
             }
         }
 
+
+        /// <summary>
+        /// Performs SRM correction on the given spectrum using the ROI and coefficients on EEPROM.
+        /// Non-ROI pixels are not corrected. If ROI or relative intensity coefficients appear
+        /// invalid, original spectrum is returned.
+        /// </summary>
+        ///
+        /// <returns>The given spectrum, with ROI srm-corrected, as an array of doubles</returns>
+        /// 
+        internal double[] correctRamanIntensity(double[] spectrum)

```

Comment:
I would consider making this public, in case customers wanted to call it themselves, for instance to correct historical spectra they'd loaded from files etc.  Your call.

Reply:
I agree that that makes sense, it also might help to address the question of when to apply by just pushing that down to the application